src/react_native/reanimated.cljs

```diff
@@ -137,3 +139,18 @@
                                  (js-obj "mass"      mass
                                          "damping"   damping
                                          "stiffness" stiffness))))
+
+(defn animate-shared-value-with-decay
+  [anim velocity clamp]
+  (set-shared-value anim
+                    (with-decay (js-obj "velocity" velocity
+                                        "clamp"    (clj->js clamp)))))
+
+(def in-out
+  (.-inOut Easing))
+
+(defn with-timing-duration
+  [val duration]
+  (with-timing val
+               (js-obj "duration" duration
```

Comment:
`#js {...}` could be used here and in other places in this file. I find the reader macro much easier to read. The `js-obj` function is only needed if you need to pass *interleaved* keys and values as arguments according to the docstring, but this is rarely the case in Clojure.

```clojure
(js-obj "duration" 100 "easing" 200) ;; => #js {:duration 100, :easing 200}
```

Reply:
changed it to `clj->js` to be consistent with the one below also